requirements/common.txt

```diff
@@ -1,11 +1,19 @@
+beautifulsoup4==4.6.0
+bs4==0.0.1
```

Comment:
Are these two packages required to run the shop, or are they only required to run the tests? Beautifulsoup is also included in https://github.com/awesto/django-shop/blob/master/requirements/test.txt